transport/src/main/java/io/scalecube/transport/Transport.java

```diff
@@ -141,17 +165,18 @@ public static Transport bindAwait(TransportConfig config) {
     address = Address.createLocal(bindPort);
 
     ServerBootstrap server = bootstrapFactory.serverBootstrap().childHandler(incomingChannelInitializer);
-    ChannelFuture bindFuture = server.bind(address.host(), address.port());
+    ChannelFuture bindFuture = server.bind(listenAddress, address.port());
```

Comment:
here should be address.host() as it was, while address.host() should contain listen address